src/Features/CSharp/Portable/UseExpressionBodyForLambda/UseExpressionBodyForLambdaCodeRefactoringProvider.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeRefactorings;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Options;
+
+namespace Microsoft.CodeAnalysis.CSharp.UseExpressionBodyForLambda
+{
+    using static UseExpressionBodyForLambdaHelpers;
+
+    [ExportCodeRefactoringProvider(LanguageNames.CSharp,
+        Name = PredefinedCodeRefactoringProviderNames.UseExpressionBody), Shared]
+    internal class UseExpressionBodyForLambdaCodeRefactoringProvider : CodeRefactoringProvider
+    {
+        public UseExpressionBodyForLambdaCodeRefactoringProvider()
```

Comment:
>UseExpressionBodyForLambdaCodeRefactoringProvider [](start = 15, length = 49)

Do we need the empty constructor?

Reply:
I'm ok with htem.  Def answers the question: where is the constructor, and what does it do?